src/main/resources/db_init.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash  -e
+
+for f in ./db/migration/V1__Base_version.sql \
```

Comment:
Maybe try to find a way to run *all* the scripts in a folder, so we don't have to update this file when adding a migration. 
At some point we would also have to change `V1...` to `V01...` (or similar) to keep them in order.

Reply:
Seems like you can do something like 
`for f in ./db/migration/*.sql; do`